.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           egress-policy: audit
 
       - name: Checkout
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v4
```

Comment:
If we are going form 4.2.2 to 5.0.0 maybe we should pull `actions/checkout@v5` or update the comment that says `v4` either is slick

Reply:
Interesting that dependabot doesn't update the comment as well.

The comment should match the version for sure. We use version pinning on SHA instead of tags because tags are mutable. I believed the comment was used to tell dependabot which major version it should track, I might have been wrong.

Reply:
Ya it seems to do that in other places so I'm not really sure why it has decided not to do so here 🤷